astroquery/linelists/cdms/core.py

```diff
@@ -8,14 +8,14 @@
 from astropy import table
 from astropy.io import ascii
 from astroquery.query import BaseQuery
-from astroquery.utils import async_to_sync
 # import configurable items declared in __init__.py
 from astroquery.linelists.cdms import conf
 from astroquery.exceptions import InvalidQueryError, EmptyResponseError
+from ..core import parse_letternumber
+from astroquery.utils import process_asyncs
```

Comment:
is this really async?

Reply:
it's used for the docs processing

Reply:
yes, I'm sorry, I was just bogged down on the non-asyncness of our async methods and the whole business of having practically duplicate methods around.